web/models/stripe_account.ex

```diff
@@ -0,0 +1,38 @@
+defmodule CodeCorps.StripeAccount do
+  @moduledoc """
+  Represents a StripeAccount stored on Code Corps.
+  """
+
+  use CodeCorps.Web, :model
+
+  @primary_key {:id, :string, []} # set primary key as string
+  @derive {Phoenix.Param, key: :id} # set method of determining primary key for route
+  schema "stripe_account" do
+    field :business_name, :string
```

Comment:
Yes, the idea here is that the business name might be logically separate from whatever the project or organization name is. For example, Code Corps is really "Code Corps PBC", but that sounds like a peanut butter sandwich and isn't super pretty to display.

We really want the business name, as well.